cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -22,8 +23,7 @@ describe('Top toolbar tests.', function() {
 	});
 
 	it('Apply highlight color.', function() {
-		cy.get('#tb_editbar_item_backcolor')
-			.click();
+		desktopHelper.actionOnSelector('backColor', (selector) => { cy.get(selector).click(); });
```

Comment:
now need to rename this file as now it doesn't contain only top toolbar tests

Reply:
> now need to rename this file as now it doesn't contain only top toolbar tests

it contains all top toolbar functionality only

Reply:
but it also tests corresponding notebook bar functionality and top toolbar is classic mode specific.